Getting Started: fastapi (#7836)
getsentry/sentry-docs: src/platforms/python/guides/fastapi/index.mdx

## Configure

> If you have the `fastapi` package in your dependencies, the FastAPI integration will be enabled automatically. There is nothing to do for you except initializing the Sentry SDK.

Reviewer: ```suggestion
If you have the `fastapi` package in your dependencies, the FastAPI integration will be enabled automatically when you initialize the Sentry SDK.
```

## Verify

```python
from fastapi import FastAPI

sentry_sdk.init(...)  # same as above

app = FastAPI()

@app.get("/sentry-debug")
async def trigger_error():
    division_by_zero = 1 / 0
```

> When you point your browser to [http://localhost:8000/sentry-debug](http://localhost:8000/sentry-debug) a transaction in the Performance section of [sentry.io](https://sentry.io) will be created. Additionally an error event will be sent to [sentry.io](https://sentry.io) and will be connected to the transaction.

Reviewer: ```suggestion
When you point your browser to [http://localhost:8000/sentry-debug](http://localhost:8000/sentry-debug) a transaction will be created in the Performance section of [sentry.io](https://sentry.io). Additionally, an error event will be sent to [sentry.io](https://sentry.io) and will be connected to the transaction.
```

</Note>

## Options

> By adding `FastApiIntegration` explicitly to your `sentry_sdk.init()` call you can set options for `FastApiIntegration` to change its behavior.

Reviewer: ```suggestion
By adding `FastApiIntegration` to your `sentry_sdk.init()` call explicitly, you can set options for `FastApiIntegration` to change its behavior.
```

Because FastAPI is based on the Starlette framework, both integrations, `StarletteIntegration` and `FastApiIntegration`, must be instantiated.

```python
from sentry_sdk.integrations.starlette import StarletteIntegration
from sentry_sdk.integrations.fastapi import FastApiIntegration

sentry_sdk.init(
    dsn="___PUBLIC_DSN___",
    # Set traces_sample_rate to 1.0 to capture 100%
    # of transactions for performance monitoring.
    traces_sample_rate=1.0,
    integrations=[
        StarletteIntegration(transaction_style="endpoint",),
        FastApiIntegration(transaction_style="endpoint",),
    ]
)
```

You can pass the following keyword arguments to `StarletteIntegration()` and `FastApiIntegration()`:

- `transaction_style`:

>   With this option, you can influence how the transactions are named in Sentry. For example:

Reviewer: ```suggestion
  This option lets you influence how the transactions are named in Sentry. For example:
```
